WorkloadManagementSystem/Client/PilotAgentsClient.py

```diff
@@ -0,0 +1,40 @@
+""" Class that contains client access to the PilotAgents handler. """
+########################################################################
+# $Id$
+# $HeadURL$
+########################################################################
+__RCSID__ = "$Id$"
+
+from DIRAC.Core.Base.Client                         import Client
+
+class PilotAgentsClient( Client ):
```

Comment:
This is another minor comment: I believe a better name would be "PilotsLoggingClient", as "PilotAgents" is an obsolete naming